src/test/java/com/currencyexchange/currency/CurrencyControllerIntegrationTest.java

```diff
@@ -0,0 +1,64 @@
+package com.currencyexchange.currency;
```

Comment:
Let`s separate tests for business logic, persistence (based on the main structure), integration etc.